src/_P039_Thermosensors.ino

```diff
@@ -1677,17 +1882,17 @@ void transfer_n_ByteSPI(uint8_t l_CS_pin_no, uint8_t l_noBytesToSend, uint8_t* l
 
   # ifndef BUILD_NO_DEBUG
 
-    if (loglevelActiveFor(LOG_LEVEL_DEBUG))
+    if (loglevelActiveFor(LOG_LEVEL_DEBUG_MORE))
     {
       String log;
       if((log.reserve(66u))) {
```

Comment:
A lot of lines reserve exactly 66 bytes for the log.
I don't think they all have exactly 66 characters in the log entry...

Reply:
This is solved with push 5853497